style-guide/styles/wds-mixins/_button-group-theming.scss

```diff
@@ -0,0 +1,27 @@
+@mixin button-group-theming($color-border) {
+	> .wds-button {
+		&:not(.wds-is-secondary) {
+			&:not(:last-child) {
+				border-right: 1px solid $color-border;
```

Comment:
so now we have to always theme the button/group buttons? shouldn't we have default colors?